scripts/skip-upgrade-tests.txt

```diff
@@ -59,4 +59,6 @@ vcd.TestAccVcdVdcDatasource.tf v2.7.0 "pVdc read did not work in 10.1"
 vcd.TestAccVcdVmInternalDisk-CreateALl.tf v2.7.0 "pVdc read did not work in 10.1"
 vcd.TestAccVcdVmInternalDisk-Update1.tf v2.7.0 "pVdc read did not work in 10.1"
 vcd.runVappNetworkTest.tf v2.7.0 "Fixed bug in DNS default"
-vcd.TestAccVcdVAppVmMultiNIC.tf v2.7.0 "Network read error in 10.1"
\ No newline at end of file
+vcd.TestAccVcdVAppVmMultiNIC.tf v2.7.0 "Network read error in 10.1"
+vcd.TestAccVcdVappNetworkDS.tf v2.8.0 "nat_enabled and firewall_enabled were moved to respective resources vcd_vapp_nat_rules and vcd_vapp_firewall_rules"
+vcd.TestAccVcdVappNetwork_Nat.tf v2.8.0 "nat_enabled and firewall_enabled were moved to respective resources vcd_vapp_nat_rules and vcd_vapp_firewall_rules"
```

Comment:
Looks like these files are also generated using `make test-binary-prepare` and they would fail when not skipped.

```
vcd.TestAccVcdVappNetwork_Isolated-Update.tf:81:  firewall_enabled      = "false"
vcd.TestAccVcdVappNetwork_Nat-Update.tf:81:  firewall_enabled      = "true"
```